### ISSUE TITLE
Calendar Week Numbers - Feature Request

### ISSUE THREAD

**Issue by MRC38** [bryntum/support#9664]
Can you consider adding a Week Number row to the calendar on the Gantt chart, so that for the start of the financial year the user could set that as Week 1?

**Comment by matsbryntse** [comment]
Something like you see here? https://bryntum.com/products/gantt/examples/custom-rendering/

**Comment by MRC38** [comment]
Yes, is there a UI for the user to set Week 1 date?

**Comment by matsbryntse** [comment]
Nope, but sounds like a reasonable feature request - thanks for contributing!
